### DISCUSSION TITLE
How to set Volume widget 'volume_down_command' key?

### DISCUSSION THREAD

**Discussion by jacobwrestlesangels** [qtile/qtile#2964] [Q&A] [answered]
Hello, I am new to qtile and loving it so far! But I am having some problems configuring the volume widget to decrease or increase the volume. I tried setting it to "F11" as that is the volume key on my laptop and it is not working. Am I misunderstanding how this key is supposed to be set?

**Comment by m-col** [comment, marked as answer]
Please post how you have configured your keybinding

**Reply by elParaguayo** [reply]
If that's the volume key, you may want to bind `XF86AudioLowerVolume` instead. Using `xev` will help show which key you actualy want to bind.

**Reply by jacobwrestlesangels** [reply]
The line I was having trouble with was `widget.Volume(volume_down_command="F11")` but inspired by the most recent reply I made an .xbindskeysrc file and added keybinds there to call amixer terminal commands and it's working perfectly! Thanks for the quick replies all! :)

**Reply by elParaguayo** [reply]
The `volume_down_command` allows you to set a custom command to decrease the volume. The default value will try to use `amixer` to set your volume.

You can then use the widget to decrease the volume by:
- Using mouse scroll (down) on the widget
- Binding a key to `lazy.widget["volume"].decrease_vol()`

**Reply by jacobwrestlesangels** [reply]
Ah I see now, that'll come in handy on my desktop thanks
